submissions/annymax/hooli-style-popup/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="css/style.css" rel="stylesheet" type="text/css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hooli Popup menu</title>
+  </head>
+  <body class="body">
+    <div class="wrapper">
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-list-item">
+            <a class="nav-link" href="">Mail</a>
+          </li>
+          <li class="nav-list-item">
+            <a class="nav-link" href="">Images</a>
+          </li>
+          <li class="nav-list-item">
+            <input class="input-popup" type="checkbox" id="button-popup" />
+            <label class="label-popup" for="button-popup">
+              <img src="icons/popup-button.png" alt="" class="img-popup" />
+            </label>
+            <div class="menu-wrapper">
```

Comment:
Please fix the size of your popup it's hard to read the content inside.
![image](https://user-images.githubusercontent.com/16196199/187023610-6ef9296d-15ef-498e-be47-fe6e3c898456.png)

Reply:
made bigger size of content